gen/generator.jl

```diff
@@ -0,0 +1,46 @@
+using Pkg
```

Comment:
Could you add a README to this directory that says what needs to be done to generate all the files? Is it enough to just run this file?

Reply:
Yes, in both terminal or vscode. Maybe a comment at the top of the file is enough.

Reply:
Yeah, if that is all there is to it, sure.